core/src/main/java/io/questdb/cairo/wal/WalWriter.java

```diff
@@ -1415,9 +1461,9 @@ public void addColumn(
                     if (uncommittedRows > 0) {
                         setColumnNull(columnType, columnIndex, segmentRowCount);
                     }
-                    LOG.info().$("added column to wal [path=").$(path).$(Files.SEPARATOR).$(segmentId).$(", columnName=").$(columnName).I$();
+                    LOG.info().$("added column to WAL [path=").$(path).$(Files.SEPARATOR).$(segmentId).$(", columnName=").$(columnName).I$();
                 } else {
-                    throw CairoException.critical(0).put("column '").put(columnName).put("' added, cannot commit because of concurrent table definition change ");
+                    throw CairoException.critical(0).put("column '").put(columnName).put("' added, cannot commit because of concurrent table definition change");
```

Comment:
this error message is confusing. Column "added" (should be "is added" or "was added") but "cannot commit" - was it added or was it not ?

The message is read by robot on your phone at night - may be we can come up with something succinct here?

(i know this might not be to do with this PR, but since we are here, we should improve message).

Reply:
The error message is the same as in the upstream branch, I'll change it to a more meaningful one.

Reply:
Renamed in 0da96063674df9fb51bacc334a93b9d4971f1a16